src/main/webapp/jsp/instructorFeedbackResultsTop.jsp

```diff
@@ -14,8 +14,13 @@
       showAll="${showAll}" />
 </c:if>
 <br>
+<div>
 <t:statusMessage statusMessagesToUser="${data.statusMessagesToUser}" />
-<br>
+</div>
```

Comment:
@whipermr5 It looks like remnant from a previous PR on this same issue. It was added to provide inline style and adjust margins. Can be removed, unless I need to do some inline CSS adjustments again.
For the other comment above, I will provide an analysis sooner.